lib/improver/tests/uv_index/test_uv_index.py

```diff
@@ -49,20 +49,25 @@ def setUp(self):
         and also one with different units."""
         data_up = np.array([[0.2, 0.2, 0.2], [0.2, 0.2, 0.2]],
                            dtype=np.float32)
+        uv_up_name = 'surface_upwelling_ultraviolet_flux_in_air'
         self.cube_uv_up = set_up_variable_cube(data_up,
-                                               name='radiation flux in '
-                                               'UV upward at surface',
-                                               units='W/m^2')
+                                               name=uv_up_name,
+                                               units='W m-2')
+        self.cube_up_badname = set_up_variable_cube(data_up,
+                                                    name='Wrong name',
+                                                    units='m')
```

Comment:
As a small comment, we probably either don't need to specify the units here, or we could specify the units as the intended units i.e. `W m-2`, as we're not using the units of the `cube_up_badname` in testing.

Reply:
Yes, sorry copy and pasting error. Now fixed.